components/bl-alert-component/styles/index.less

```diff
@@ -0,0 +1,189 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-alert-fontFamily: "Roboto","Helvetica","Arial",sans-serif;
+@bl-customComponent-alert-fontSize: 0.875rem;
```

Comment:
could we use `px`? i think there could be some collisions if we put this component into some container with a defined `font-size`